python-avd/pyavd/_cv/client/studio.py

```diff
@@ -322,8 +305,7 @@ async def get_studio_inputs_with_path(
         try:
             response = await client.get_one(request, metadata=self._metadata, timeout=timeout)
         except Exception as e:  # pylint: disable=broad-exception-caught
```

Comment:
Since you are only interested in this type, we can improve this:
```suggestion
        except GRPCError as e:
```
And then just check the status.not_found.

Reply:
why do you need the broad exception here? Shouldn't that be handled by the decorator?